marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/keystore/AzureKeyStoreInit.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.keystore;
+
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.io.InputStream;
+
+@Component
+@Profile("azure")
+public class AzureKeyStoreInit extends AbstractKeyStoreInit {
+
+    @Value("${key-store-password}")
+    private String keyStorePassword;
+
+    @Value("${azure.storage.endpoint}")
+    private String endpoint;
+
+    @Value("${azure.storage.container.name}")
```

Comment:
Just do "containerName" and "blobName", unless we think we're going to have other properties on objects named "container" and "blob".

Reply:
This is to maintain naming consistency with the spring property (`azure.keyvault.uri`) added by `azure-keyvault-secrets-spring-boot-starter` dependency.
Plus it allows us to easily identify the cloud platform and the cloud service that the property is being used for.

Reply:
The question is whether the "container" should be an object for property-naming purposes. If the property name on "storage" is "container name", then we should use "containerName" - we don't want to use a period as a substitute for a space. But if "container" is essentially a structured property of "storage" and "name" is just one property of "container", then "container.name" is correct. Same logic applies for "bucket".

Basically - we want to use a period to indicate that something is conceptually an object, not as a substitute for a space.

Reply:
Fair enough. I'll change this to `azure.storage.containerName` and make similar updates to other properties.